docs/apis-tools/operate-api/authentication.md

```diff
@@ -4,57 +4,141 @@ title: Authentication
+:::
 
-The following settings are needed to request a token:
+</TabItem>
 
-| Name                     | Description                                     | Default value        |
-| ------------------------ | ----------------------------------------------- | -------------------- |
-| client id                | Name of your registered client                  | -                    |
-| client secret            | Password for your registered client             | -                    |
-| audience                 | Permission name; if not given use default value | `operate.camunda.io` |
-| authorization server url | Token issuer server                             | -                    |
+<TabItem value='self-managed'>
 
-:::note
-For more information on how to get these values for Camunda 8, read [Manage API Clients](/components/console/manage-clusters/manage-api-clients.md).
+:::tip
+The URL of the Operate REST API, represented below by the `${CAMUNDA_OPERATE_BASE_URL}` variable, is configured in your Self-Managed installation. The default value is `http://localhost:8081`.
 :::
 
-Send a token issue _POST_ request to the authorization server with the required settings:
+</TabItem>
+
+</Tabs>
 
 ```shell
-curl -X POST -H 'content-type: application/json' -d '{"client_id": "RgVdPv...", "client_secret":"eDS1~Hg...","audience":"operate.camunda.io","grant_type":"client_credentials"}' https://login.cloud.camunda.io/oauth/token
+curl --request POST ${CAMUNDA_OPERATE_BASE_URL}/v1/process-instances/search \
+   --header "Authorization: Bearer ${TOKEN}" \
+   --header 'Content-Type: application/json' \
+   --data-raw '{}'
 ```
 
-You will get something like the following:
+A successful response would include [matching process instances](./specifications/search-1.api.mdx). For example:
 
 ```json
 {
-  "access_token": "eyJhbG...",
-  "scope": "f408ca38-....",
-  "expires_in": 58847,
-  "token_type": "Bearer"
+  "items": [],
+  "sortValues": [123456],
+  "total": 0
 }
 ```
 
-Capture the `access_token` value from the response object. In each request to the Operate API, include it as an authorization header:
-
-```
-Authorization: Bearer eyJHb...
-```
-
-## Authentication for Self-Managed cluster
-
-### Authentication via Identity JWT access token
+## Token expiration
 
-This authentication method is described in [Operate Configuration - Authentication](/self-managed/operate-deployment/operate-authentication.md#identity).
+Access tokens expire according to the `expires_in` property of a successful authentication response. After this duration, in seconds, a new access token must be requested.
 
-### Authentication via cookie
+## Authentication via cookie (Self-Managed only)
 
 Another way to access the Operate API in a Self-Managed cluster is to send cookie headers in each request. The cookie can be obtained by using the API endpoint `/api/login`. Take the steps in the following example:
```

Comment:
NBC: Change to

You can also access the Operate API in a Self-Managed cluster by sending cookie headers in each request. You can obtain a cookie using the `/api/login` API endpoint. For example:

Reply:
```suggestion
You can also access the Operate API in a Self-Managed cluster by sending cookie headers in each request. You can obtain a cookie using the /api/login API endpoint. For example:
```